.fides/db_dataset.yml

```diff
@@ -1470,3 +1470,43 @@ dataset:
       data_categories:
       - system.operations
       data_qualifier: aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified
+  - name: userregistration
+    description: 'Records the registration status of this Fides deployment'
+    data_categories: null
+    data_qualifier: aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified
+    fields:
+    - name: id
+      description: 'Fides Generated Description for Column: id'
```

Comment:
Can we polish these descriptions slightly and replace the auto-generated ones?

Reply:
We ought to do this wholesale for the entire file, have made a [follow-up ticket](https://github.com/ethyca/fides/issues/1606).